Docs/source/pages/advanced-usage.md

```diff
@@ -539,6 +539,17 @@ Logging on the driver is handled using `NLog` in C#, `loguru` in python and `log
 
 ```
 
+## Logging in WebGL
```

Comment:
This section should not be included on the `development` branch just yet and for this hotfix, only on the `feature-proxy` branch